src/scss/_variables.scss

```diff
@@ -16,11 +16,8 @@ $o-loading-animation-keyframes: false !default;
 
 /// Predefined color themes
 ///
-/// @type Map
-$o-loading-colors: (
-	'light': oColorsGetPaletteColor('white'),
-	'dark': oColorsGetPaletteColor('black-80')
-);
+/// @type List
+$o-loading-colors: ('light', 'dark');
```

Comment:
Does changing the data type from Map to List make this a breaking change?

Reply:
Indeed you are technically correct. Making a change now.